src/static/js/modules/init-chosen.js

```diff
@@ -10,7 +10,9 @@ require( 'chosen' );
 function init() {
   $( '.chosen-select' ).chosen( {
     width:           '100%',
+    /* eslint-disable */
```

Comment:
Went with your second one so I'm only adding one additional warning to suppress 4ebb5d0